include/engine/search_engine_data.hpp

```diff
@@ -23,6 +24,21 @@ struct ManyToManyHeapData : HeapData
     ManyToManyHeapData(NodeID p, EdgeWeight duration) : HeapData(p), duration(duration) {}
 };
 
+struct MultiLayerDijkstraHeapData : HeapData
```

Comment:
I have tested performance of creating a new heap and clearing, it is almost the same. 
It should be changed in #3816